+dj/Schema.m

```diff
@@ -32,6 +32,7 @@
         %   computed: tableName with '__'
         allowedTiers = {'lookup' 'manual' 'imported' 'computed' 'job'}
         tierPrefixes = {'#', '', '_', '__', '~'}
+        tierClasses = {'dj.Lookup' 'dj.Manual' 'dj.Imported' 'dj.Computed' 'dj.Jobs'}
```

Comment:
I agree but I think we can hold off with that since it modifies existing functions.  I am working on unit regression tests and changes like these will be easier to make.